content/docs/user-guide/related-technologies.md

```diff
@@ -0,0 +1,134 @@
+# Comparison with Related Technologies
+
+DVC combines a number of existing ideas into a single tool, with the goal of
+bringing best practices from software engineering into the data science field
+(refer to [What is DVC?](/doc/user-guide/what-is-dvc) for more details).
+
+### Git
+
+- DVC builds upon Git by introducing the concept of
+  [data files](/doc/user-guide/basic-concepts#data-files) – large files that
+  should not be stored in a Git repository, but still need to be tracked and
+  versioned. It leverages Git's features to enable managing different versions
+  of data itself, data pipelines, and experiments.
+
+- DVC is not fundamentally bound to Git, and can work without it (except
+  versioning-related features). This also applies to Git-LFS and Git-annex,
+  below.
+
+### Git-LFS (Large File Storage)
+
+- DVC does not require special servers like Git-LFS demands. Any cloud storage
+  like S3, Google Cloud Storage, or even an SSH server can be used as a
+  [remote storage](/doc/command-reference/remote). No additional databases,
+  servers, or infrastructure are required.
+
+- DVC does not add any hooks to the Git repo by default (although they are
+  [available](/doc/command-reference/install)).
+
+- Git-LFS was not made with data science in mind, so it doesn't provide related
+  features (e.g. [pipelines](/doc/user-guide/basic-concepts#data-pipeline),
+  [metrics](/doc/command-reference/metrics), etc.).
+
+- Github (most common Git hosting service) has a limit of 2 GB per repository.
+
+### Git-annex
+
+- DVC can use reflinks\* or hardlinks (depending on the system) instead of
+  symlinks to improve performance and the user experience.
+
+- Git-annex is a datafile-centric system whereas DVC focuses on providing a
+  workflow for machine learning and reproducible experiments. When a DVC or
+  Git-annex repository is cloned via `git clone`, data files won't be copied to
+  the local machine, as file contents are stored in separate
+  [remotes](/doc/command-reference/remote). With DVC however, `.dvc` files,
+  which provide the reproducible workflow, are always included in the Git
+  repository. Hence, they can be executed locally with minimal effort.
+
+- DVC optimizes file hash calculation.
```

Comment:
How is this important when comparing with Git-annex?

Reply:
hard to tell, let's keep it as is for now ... to be honest not sure that related tech deserves a section ... would be good to make it short and part of the FAQ at some point

Reply:
Well, it's a pretty long page.

Reply:
the section it's not that useful the way it's done now, too complicated sometimes, and not comprehensive

yet another page for us to support.

We can keep it somewhere for now, but it should be rewritten to be more useful at some point.

Reply:
I could add some links to it from a few places (currently there's only one from https://dvc.org/doc/use-cases/versioning-data-and-model-files) and remove it from the nav for now. Lmk

Reply:
if we want to remove something from the nav it almost 100% it should be removed completely. 

Adding links - there should be a good reason for that. Not only because we want to keep the page.

Let's just keep it as is more or less for now. Not an important one, for sure.